tests/functional/tests/test_prometheus_blackbox_exporter.py

```diff
@@ -59,14 +57,9 @@ def test_01_api_ready(self):
     def test_02_nrpe_http_check(self):
         """Verify nrpe check exists."""
         expected_nrpe_check = (
-            "command[check_prometheus_blackbox_exporter_http]={} -I 127.0.0.1 "
-            "-p {} -u {}"
-        ).format(
-            "/usr/lib/nagios/plugins/check_http", DEFAULT_API_PORT, DEFAULT_API_URL
-        )
-        logging.debug(
-            "Verify the nrpe check is created and has the required content..."
-        )
+            "command[check_prometheus_blackbox_exporter_http]={} -I 127.0.0.1 " "-p {} -u {}"
```

Comment:
Just a nit, the lines can been joined without joining the string:

```suggestion
            "command[check_prometheus_blackbox_exporter_http]={} -I 127.0.0.1 -p {} -u {}"
```